src/main/java/frc/robot/subsystems/ClimbSubsystem.java

```diff
@@ -0,0 +1,19 @@
+package frc.robot.subsystems;
+
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+public class ClimbSubsystem extends SubsystemBase{
+    PivotSubsystem pivot;
+    ElevatorSubsystem elevator;
```

Comment:
These should be initialized in the constructor

Reply:
Wait this is a subsystem, how would you fit it in the constructor? Unless you want me to do the, this.pivot = pivot;